networker/networker_test.go

```diff
@@ -1191,6 +1201,7 @@ func TestCombined_Reconnect(t *testing.T) {
 				config.NewAllowlist(nil, nil, nil),
 				[]string{"1.1.1.1"},
 				test.enableLan,
+				func(startTime time.Time, success bool, err error) {},
```

Comment:
[nit] how do you feel about defining here in this module, a variable that would hold this empty callback - and simply reuse if for all the tests?